examples/platforms/cc2538/radio.c

```diff
@@ -704,6 +716,8 @@ static void readFrame(void)
     {
         sReceiveFrame.mLength            = length;
         sReceiveFrame.mInfo.mRxInfo.mLqi = crcCorr & CC2538_LQI_BIT_MASK;
+        // sSrcMatchFound should have been set if this frame is acked with FP
+        sReceiveFrame.mInfo.mRxInfo.mAckedWithFramePending = cc2538SrcMatchEnabled() ? sSrcMatchFound : true;
```

Comment:
Is it possible to directly set `mAckedWithFramePending` during the src_matched interrupt?

Reply:
No, the `SRC_MATCH_FOUND` interrupt comes before `RX` interrupt, so at that time, even `readFrame` has not executed, and there is no data in `sReceiveFrame`

Reply:
Maybe yes, need more thinking

Reply:
Setting `sReceiveFrame.mInfo.mRxInfo.mAckedWithFramePending` or `sSrcMatchFound` has no real difference because they are both static variables. Would be more understandable using a separate variable?